src/components/TaskRunListView/TaskRunListView.tsx

```diff
@@ -38,6 +49,16 @@ const TaskRunListView: React.FC<React.PropsWithChildren<Props>> = ({ taskRuns, l
     [nameFilter, sortedTaskRuns],
   );
 
+  // Handle error state
+  if (error) {
```

Comment:
we can use `getErrorState` utility :)

Reply:
Nice idea. 
Updated for the awesome utils. :-) .